packages/markdown/src/plugins/assetsPlugin/assetsPlugin.ts

```diff
@@ -13,6 +13,17 @@ export interface AssetsPluginOptions {
    * Prefix to add to relative assets links
    */
   relativePathPrefix?: string
+
+  /**
+   * The strictness of the determination of relative paths
+   *
+   * When different options are set, the criteria for identifying relative paths are as follows:
+   *
+   * - true:       start with `./` or `../`
+   * - false:      no start with `/` or `<protocol header>`
+   * - '@-perfix': no start with `/` or `@` or `<protocol header>`
+   */
+  restrictRelativePath?: boolean | '@-perfix'
```

Comment:
I am not sure about the option name, but for things like `@-prefix` or the previous `half`, I think we are mainly talking about alias behaviors and its influence with relative paths, rather than path support and how it affects alias.

If so, the option name shall be something related to alias like my original comment.